webapp-django/crashstats/crashstats/views.py

```diff
@@ -2545,7 +2515,7 @@ def gccrashes(request, product, version=None, default_context=None):
         # No version was passed get the latest nightly
         version = get_latest_nightly(context, product)
 
-    current_products = context['currentproducts']['products']
+    current_products = context['active_versions'].keys()
```

Comment:
Because `context['active_versions']` is an `OrderedDict` object, this list of keys will thus forth be in the magical order as dictated by this SQL: https://github.com/mozilla/socorro/blob/master/socorro/external/postgresql/products.py#L104
which ultimately comes from this: https://github.com/mozilla/socorro/blob/master/socorro/external/postgresql/raw_sql/views/001_product_info_view.sql#L12